doc/source/ray-observability/ray-metrics.rst

```diff
@@ -41,30 +41,58 @@ First, download Prometheus. `Download Link <https://prometheus.io/download/>`_
     tar xvfz prometheus-*.tar.gz
     cd prometheus-*
 
-Let's modify Prometheus's config file to scrape metrics from Prometheus endpoints.
+Ray provides a prometheus config that works out of the box. After running ray, it can be found at `/tmp/ray/metrics/prometheus/prometheus.yml`.
```

Comment:
Can you mention this is available only in `ray[default]`?

Reply:
do we also record this per ray session? ie why not scope it to 'tmp/ray/session_latest"?

Reply:
yes, changed to be session scoped. Will need to update `prom_metrics_service_discovery.json` as well in a future PR.